File: gcc/1.c
#include <assert.h>
#include <stdio.h>

int main()
{
  int sum = 0;
  int i = 0;
  for (i = 0; i < 1000; i++) {
    if (i % 3 == 0) {
      sum += i;
    } else if (i % 5 == 0) {
      sum += i;
    }
  }

  printf("%d\n", sum);
  assert(sum == 233168);
  return 0;
}

File: solns/csharp/1.cs
using System;
using System.Diagnostics;
using System.Linq;


var ns = from n in Enumerable.Range(1, 999)
         where n % 3 == 0 || n % 5 == 0
         select n;
var result = ns.Sum();

Console.WriteLine(result);
Trace.Assert(result == 233168);
